cli/src/main/java/com/box/l10n/mojito/cli/apiclient/WsApiConfig.java

```diff
@@ -0,0 +1,55 @@
+package com.box.l10n.mojito.cli.apiclient;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class WsApiConfig {
```

Comment:
Why are these new Beans required? Why can't we use the existing *WS.java class endpoints?